nullaway/src/test/java/com/uber/nullaway/NullAwaySerializationTest.java

```diff
@@ -1414,6 +1451,49 @@ public void errorSerializationTestLocalClassField() {
         .doTest();
   }
 
+  @Test
+  public void verifySerializationVersionIsSerialized() {
```

Comment:
Can we make a helper method for this test that takes the desired version number and then two one-liner test cases that call it for 1 and 2?

Reply:
Sure, fad05620a6e5365c25e2eb138847e2b805c13959